src/disc/Albany_ConnManager.hpp

```diff
@@ -46,6 +46,15 @@ class ConnManager : public panzer::ConnManager {
 
   // Queries the dimension of a part
   virtual int part_dim (const std::string& part_name) const = 0;
+  
+  /** Get vector of bools associated to connectivity for a particular element, indicating whether the entity is owned by this rank
+    *
+    * \param[in] localElmtId Local element ID
+    *
+    * \returns Pointer to beginning of bools, with total size
+    *          equal to <code>getConnectivitySize(localElmtId)</code>
+    */
+  virtual const std::vector<bool>& getOwnership(LO localElmtId) const = 0;
```

Comment:
`std::vector<bool>` is a dangerous type. Iterators and access members are different from a normal `std::vector`. I've had my share of bad surprises with it, so I would use either a `std::vector<int>`, `std::vector<char>`, or `std::deque<bool>`.

Reply:
Yeah, in fact it gave some troubles because `&m_ownership[i]` is not ` bool*`. I think the way I'm using it now is safe, see the reply to your comment in `STKConnManager.hpp`. However, feel free to change it.